--- ghemical-0.90/openbabel/c3d.cpp.orig	Wed Jun 19 23:18:59 2002
+++ ghemical-0.90/openbabel/c3d.cpp	Fri Jun 27 14:43:16 2003
@@ -150,7 +150,7 @@
     }
 
   if (!natoms) return(false);
-  divisor = pow(10.0,exponent);
+  divisor = pow(10.0f,exponent);
   mol.ReserveAtoms(natoms);
 
   ttab.SetToType("INT");
--- ghemical-0.90/src/graphics/views.cpp.orig	Thu Jun 20 13:27:04 2002
+++ ghemical-0.90/src/graphics/views.cpp	Fri Jun 27 15:11:23 2003
@@ -486,7 +486,7 @@
 
 	glInitNames();
 	
-	i32s zzz = (i32s) sqrt(dv.size());	// support square datasets only!!!
+	i32s zzz = (i32s) sqrt((double)dv.size());	// support square datasets only!!!
 	for (i32s n1 = 0;n1 < zzz - 1;n1++)
 	{
 		for (i32s n2 = 0;n2 < zzz - 1;n2++)
@@ -542,7 +542,7 @@
 	if (sy < 0.0) return;
 	if (sy > 1.0) return;
 	
-	i32s zzz = (i32s) sqrt(dv.size());	// support square datasets only!!!
+	i32s zzz = (i32s) sqrt((double)dv.size());	// support square datasets only!!!
 	i32s index = (i32s) (((i32s) (sx * zzz)) * zzz + sy * zzz);
 	if (index >= (i32s) dv.size()) return;
 	
