--- ghemical-2.01/configure.ac.orig	2006-04-24 14:06:32.000000000 +0200
+++ ghemical-2.01/configure.ac	2006-07-23 22:48:39.386085750 +0200
@@ -123,10 +123,6 @@
 AC_CHECK_LIB([GL], [glMatrixMode])
 AC_CHECK_LIB([GLU], [main])
 AC_CHECK_LIB([glut], [glutInit])
-AC_CHECK_LIB([X11], [main])
-AC_CHECK_LIB([Xmu], [main])
-AC_CHECK_LIB([Xext], [main])
-AC_CHECK_LIB([Xi], [main])
 
 # Checks for header files.
 # ^^^^^^^^^^^^^^^^^^^^^^^^
