--- ghemical-1.01/Makefile.in~	2002-12-03 07:44:52.000000000 +0000
+++ ghemical-1.01/Makefile.in	2005-06-12 00:37:10.564721376 +0000
@@ -64,26 +64,26 @@
 	@echo ""
 	@echo "Creating directories (if needed)..."
 	-@for i in $(create_dirs); do \
-		$(top_srcdir)/mkinstalldirs $$i; \
+		$(top_srcdir)/mkinstalldirs $(DESTDIR)$$i; \
 	done && test -z "$$fail"
 	@echo "Installing ghemical"
-	$(INSTALL_PROGRAM) bin/ghemical $(bindir)
+	$(INSTALL_PROGRAM) bin/ghemical $(DESTDIR)$(bindir)
 	@echo ""
 	@echo "Copying parameter files"
-	cp -r -p $(top_srcdir)/bin/parameters $(version_dest);
+	cp -r -p $(top_srcdir)/bin/parameters $(DESTDIR)$(version_dest);
 	@echo ""
 	@echo "Copying libglade data files"
-	cp -p $(top_srcdir)/src/target3/glade/*.glade $(version_dest)/glade;
+	cp -p $(top_srcdir)/src/target3/glade/*.glade $(DESTDIR)$(version_dest)/glade;
 	@echo ""
 	@echo "Copying example files"
-	cp -r -p $(top_srcdir)/bin/examples $(data_dest);
+	cp -r -p $(top_srcdir)/bin/examples $(DESTDIR)$(data_dest);
 	@echo ""
 	@echo "Copying file translators"
-	cp -p $(top_srcdir)/openbabel/*.txt $(version_dest);
+	cp -p $(top_srcdir)/openbabel/*.txt $(DESTDIR)$(version_dest);
 	@echo ""
 	@echo "Copying help files"
-	cp -p $(top_srcdir)/bin/user-docs/*.html $(gnome_help);
-	cp -p $(top_srcdir)/bin/user-docs/images/*.png $(gnome_help)/images;
+	cp -p $(top_srcdir)/bin/user-docs/*.html $(DESTDIR)$(gnome_help);
+	cp -p $(top_srcdir)/bin/user-docs/images/*.png $(DESTDIR)$(gnome_help)/images;
 	
 timestamps:
 	@for i in $(SRCDIRS); do \
